src/app/loans/create-loans-account/create-loans-account.component.html

```diff
@@ -36,7 +36,7 @@
       <ng-template matStepLabel>TERMS</ng-template>
 
       <mifosx-loans-account-terms-step [loansAccountProductTemplate]="loansAccountProductTemplate"
-      [loansAccountTemplate] = "loansAccountTemplate" [multiDisburseLoan] = "multiDisburseLoan" [collateralOptions]="collateralOptions"
+      [loansAccountTemplate] = "loansAccountTemplate" [collateralOptions]="collateralOptions"
```

Comment:
@josehernandezfintecheandomx Can you help me why the multiDisburseLoan get removed?